File: vetor/exercicio1.py

notas = [0] * 10  

for i in range(10):
    notas[i] = float(input(f"Digite a nota do aluno {i+1}: "))

soma = 0
for i in range(10):
    soma += notas[i]
media = soma / 10

acima_da_media = 0
for i in range(10):
    if notas[i] > media:
        acima_da_media += 1

print(f"\nMédia da turma: {media:.2f}")
print(f"Número de alunos com nota acima da média: {acima_da_media}")

File: vetor/exercicio1.c
#include <stdio.h>

int main() {
    float notas[10];
    float soma = 0.0, media;
    int i, count = 0;

    printf("Digite as notas dos 10 alunos:\n");
    for (i = 0; i < 10; i++) {
        printf("Aluno %d: ", i + 1);
        scanf("%f", &notas[i]);
        soma += notas[i];
    }

    media = soma / 10;

    for (i = 0; i < 10; i++) {
        if (notas[i] > media) {
            count++;
        }
    }

    printf("Média da turma: %.2f\n", media);
    printf("Número de alunos com nota superior à média: %d\n", count);

    return 0;
}

